src/js/controller/model.js

```diff
@@ -151,20 +157,18 @@ const Model = function() {
                 this.persistVideoSubtitleTrack(data.currentTrack, data.tracks);
                 break;
             case 'visualQuality':
-                var visualQuality = Object.assign({}, data);
-                mediaModel.set('visualQuality', visualQuality);
+                mediaModel.set('visualQuality', Object.assign({}, data));
```

Comment:
@robwalch do we polyfill this for IE11? I'm seeing this not trigger for html5 provider

Reply:
We do. I'll look at the bug you filed.